cuda/preconditioner/jacobi_kernels.cu

```diff
@@ -277,6 +278,39 @@ __global__ void __launch_bounds__(warps_per_block *cuda_config::warp_size)
 }
 
 
+template <int max_block_size, int subwarp_size, int warps_per_block,
+          typename ValueType, typename IndexType>
+__global__ void __launch_bounds__(warps_per_block *cuda_config::warp_size)
```

Comment:
Very small nit: It looks like the `cuda_config::warp_size` is a pointer instead of a multiplication with `warps_per_block`.
```suggestion
__global__ void __launch_bounds__(warps_per_block * cuda_config::warp_size)
```

Reply:
cmake-format is the one who does that for some reason

Reply:
And I think it does think that this is a pointer, my linter also shows it exactly in that way.

Reply:
Then I guess we leave it as it is, it is definitively not a big deal, just looked a bit off to me.
I definitively don't think it is worth looking into the style settings for clang-format to change such a small detail, so I will mark this as 'Resolved'.